### ISSUE TITLE
List View: disable the animation and initial focus when the 'Always open list view' preference is enabled

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#61519]
### Description

See also https://github.com/WordPress/gutenberg/issues/61498

In WordPress 6.5 the List View panel opens without animation.
On trunk, it now opens with a slide-in animation.

When the 'Always open list view' preference is enabled, the List View is always open also on page load. However, the animation runs also when the editor page loads, which isn't a great experience for users. Things that 'move' on the screen on page load are confusing, they trigger a layout shift, and are somehow a jarring experience for users. On page load, the animation is unnecessary and should be prevented.

Re: Initial focus.
When manually opening the List View, setting initial focus within it makes sense. However, when the 'Always open list view' preference is enabled, initial focus is set on the List View on page load. This is less than ideal. It is unexpected, as the tab sequence should normally start from the document root.

ANimat4ed GIF to illustrate the animation running on page load:

![list view animation on load](https://github.com/WordPress/gutenberg/assets/1682452/faba504d-7677-4c11-8649-6ede90865264)

### Step-by-step reproduction instructions

- Go to the Post editor > Options > Preferences, and enable the 'Always open list view' preference.
- Go back to the posts list in the WordPress admin.
- Click a post to edit it.
- Observe that when the editor page loads, the List View appears with a slide-in animation.
- Observe that initial focus is set on the first item in the List View.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by SainathPoojary** [comment]
Hey @afercia,

I attempted to reproduce the issue, and it appears to be resolved now. Could you please confirm on your end as well?

https://github.com/user-attachments/assets/87eb1a8f-f5e7-4b92-b664-59f11ba6bb43

**Comment by afercia** [comment]
Thanks @SainathPoojary  for testing this issue. Yes, I can't reproduce it on current trunk any longer.
I tried to track down where it changed but I couldn't find it. Maybe it changed in https://github.com/WordPress/gutenberg/pull/60693 but I'm not sure. I'm not sure the 'fix' was intentional and it would be nice to make sure any animation is disabled when `showListViewByDefault` is true.

Regarding the Initial focus, that still occurs on trunk
